v1.14/inspur-incloud-amd64/PRODUCT.yaml

```diff
@@ -0,0 +1,9 @@
+vendor: Inspur
+name: InCloud K8S
```

Comment:
Could you please rename to K8s? See https://github.com/cncf/foundation/blob/master/style-guide.md.

Reply:
Hi, I've renamed the production name for amd64 and arm64.